src/presentation/http/fastify.d.ts

```diff
@@ -125,5 +125,23 @@ declare module 'fastify' {
      * @param message - Optional message to send. If not specified, default message will be sent
      */
     notAcceptable: (message?: string) => Promise<void>;
+
+    /**
+     * Custom method for sending 400 error
+     *
+     * Send this error when a domain-level error is thrown
+     *
+     * @example
+     *
+     *  try {
+     *    if (updatedNote === null) {
+     *      throw new DomainError(`Note with id ${id} was not updated`);
+     *  } catch (error: DomainError) {
+     *      reply.domainError(error.message);
+     * }
```

Comment:
its not a real-world example

Reply:
Can you show an example?  I don't clearly understand what the real-world example should look like

Reply:
```ts
try {
  service.someAction()
} catch (error: Error) {
  if (error instanceof DomainError) {
    reply.domainError(error.message);

    return;
  }
  
  throw error;
}
```